pillar/edx/xqwatcher_600.sls

```diff
@@ -2,6 +2,7 @@
 {% set environment = salt.grains.get('environment', 'mitx-qa') %}
 {% set env_data = env_settings.environments[environment] %}
 {% set xqwatcher_venv_base = '/edx/app/xqwatcher/venvs' %}
+{% set python3_version = '3.7' %}
```

Comment:
This will result in the Python path being `/usr/bin/3.7` rather than `/usr/bin/python3.7`